src/main/java/org/apache/sling/feature/cpconverter/index/IndexDefinitions.java

```diff
@@ -74,6 +76,11 @@ public void addNode(@NotNull String parentPath, @NotNull DocViewNode2 node) {
         if ( existing != null ) {
             currentChildren.remove(existing);
         }
+        if ( CollectionUtils.isEmpty(node.getProperties())
+                && node.getName()!= null
```

Comment:
SonarCloud says that you can remove this check, `getName()` never returns null

https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=169&id=apache_sling-org-apache-sling-feature-cpconverter&open=AYi4b0ONGxwo1pTO_aYE